com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -715,7 +649,10 @@ public List<String> getPropertyFiles() {
     public List<IResource> getPropertyFilesAsResources() {
         List<IResource> resources = new ArrayList<>();
         for (String propertyFile : propertyFiles) {
-            resources.add(fileSystem.get(propertyFile));
+            Path rootDir = Paths.get(getRootDirectory()).normalize().toAbsolutePath();
+            Path settingsFile = Paths.get(propertyFile).normalize().toAbsolutePath();
+            Path relativePath = rootDir.relativize(settingsFile);
+            resources.add(fileSystem.get(relativePath.toString()));
```

Comment:
if root folders specified, settings paths should use full path either like `root_folder` + `settings/path`. But such path can't be used by the build system, so we use relative path when we use settings files as `IResource`